common/build/eslint-config-fluid/src/custom-rules/no-restricted-tags-imports.js

```diff
@@ -0,0 +1,91 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+const { TSDocParser } = require("@microsoft/tsdoc");
+
+module.exports = {
+	meta: {
+		type: "problem",
+		docs: {
+			description:
+				"This rule restricts imports from specified tags or non-public APIs. This to prevent accidental dependencies on internal, unstable or undocumented parts of the codebase.",
+			category: "Best Practices",
+		},
+		fixable: "code",
+		schema: [
+			{
+				type: "object",
+				properties: {
+					tags: {
+						type: "array",
+						items: { type: "string" },
+						uniqueItems: true,
+					},
+					exceptions: {
+						type: "array",
+						items: { type: "string" },
+						uniqueItems: true,
+					},
+				},
```

Comment:
Should we make this into a dictionary with tags as keys and the list of exceptions for the given tag as values? So we can do something like "you can import @internal from ./foo, and @alpha from ./bar". Or can that be specified by using multiple entries in options? If so, then the functional logic of the rule probably cannot look at only `context.options[0]`, it needs to consider them all.

Reply:
Good idea, a map/dict would make it much more robust.